app/models/room.rb

```diff
@@ -1,5 +1,9 @@
 # A Room in Convene acts as a gathering place.
 class Room < ApplicationRecord
+  has_one_attached :image do |attachable|
+    attachable.variant :thumb, resize_to_limit: [150, 150]
```

Comment:
I don't see the variant being saved locally so jury is still out on this line of code.